server/index.js

```diff
@@ -115,11 +119,19 @@ module.exports = (_config = {}) => {
       if (walletClient) {
         await walletClient.open();
       }
+      if (multisigWalletClient) {
+        await multisigWalletClient.open();
+      }
     } catch (err) {
       logger.error('Error connecting sockets: ', err);
     }
 
-    bsock.on('socket', socketHandler(nodeClient, walletClient));
+    // TODO: figure out if duplicating some events between
+    // the two different wallet clients
```

Comment:
Yeah, that's a good point. Since the multisig client is just an extension of the WalletClient from `bclient` I imagine there will be some duplication. 

@nodar-chkuaselidze You have any thoughts on this?

Reply:
Going to poke once more @nodar-chkuaselidze 😁 
We actually should look into improving/overhauling the socket infrastructure overall anyway.